src/main/java/net/sf/jabref/gui/actions/CleanupAction.java

```diff
@@ -155,10 +156,10 @@ private int showDialog() {
     /**
      * Runs the cleanup on the entry and records the change.
      */
-    private void doCleanup(CleanupPreset preset, BibEntry entry, NamedCompound ce) {
+    private void doCleanup(CleanupPreset preset, BibEntry entry, BibDatabase bibDatabase, NamedCompound ce) {
         // Run cleaner
         CleanupWorker cleaner = new CleanupWorker(preset,
-                panel.getBibDatabaseContext().getMetaData().getFileDirectory(Globals.FILE_FIELD), null,
+                panel.getBibDatabaseContext().getMetaData().getFileDirectory(Globals.FILE_FIELD), bibDatabase,
```

Comment:
Or should one simply add panel.getDatabase() here? Or pass Globals.journalAbbreviationLoader.getRepository() to doCleanup() as well?

Guess it doesn't really matter...